x11-misc/xpad/xpad-5.7.0.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-inherit autotools
+inherit autotools xdg-utils
```

Comment:
You could've just inherited `xdg` here, then you wouldn't manually have to call the functions in pkg_post* phases. But this works too, thanks!